src/Compilers/Core/Portable/CodeGen/ILBuilderEmit.cs

```diff
@@ -4,6 +4,7 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using System.Linq;
```

Comment:
`using static System.Linq.ImmutableArrayExtensions` 

We generally do not want to work with `ImmutableArray` through regular Linq extensions